src/index.html

```diff
@@ -3,6 +3,8 @@
   <head>
     <meta charset="utf-8" />
     <title>OpenStreetMap Americana</title>
+    <link rel="preconnect" href="https://osm-americana.github.io" />
+    <link href="fonts.css" rel="stylesheet" />
```

Comment:
Is the https://github.com/ZeLonewolf/openstreetmap-americana/pull/955#issuecomment-1751888674 still possible, but less likely? Or should we continue to investigate a JavaScript-based alternative to the CSS stylesheet that can respond to font loading events?

Reply:
Yes still possible.  Try shield test with browser cache disabled and you'll see a fallback font for the first few rows.